postprocessing/OEP_MaStR_OPSD_Comparison.ipynb

```diff
@@ -1707,7 +1705,7 @@
     "# load renewables\n",
     "df_renewables = pd.read_csv(\n",
     "    \"data/OPSD/renewable_power_plants_DE.csv\",\n",
-    "    sep=\";\", encoding=\"ANSI\", parse_dates=[\"commissioning_date\"]\n",
+    "    sep=\";\", encoding=\"latin\", parse_dates=[\"commissioning_date\"]\n",
```

Comment:
I could not use 'ANSI' as it is only for Windows (according to [python's doc](https://docs.python.org/3/library/codecs.html#text-encodings)). Can you try to see if `'latin'` works by you too?